pkg/operator/controller/ingress/controller.go

```diff
@@ -617,6 +617,27 @@ func setDefaultPublishingStrategy(ic *operatorv1.IngressController, platformStat
 					statusLB.ProviderParameters.IBM.Protocol = specProtocol
 					changed = true
 				}
+			case operatorv1.OpenStackLoadBalancerProvider:
```

Comment:
@Miciah and reviewers: I didn't add a check here whether the scope is external (we have CEL validation) I wonder if we want it.